idea/idea-core/resources/messages/KotlinIdeaCoreBundle.properties

```diff
@@ -13,5 +10,4 @@ notification.description.text.script.configuration.has.been.changed=Load changes
 text.kotlin.loading.script.configuration=Kotlin: Loading script configuration...
 text.loading.kotlin.script.definitions=Loading kotlin script definitions
 text.loading.kotlin.script.configuration=Loading script configuration
-implement.members.handler.title=Implement Members
```

Comment:
Just move the whole KotlinIdeaCoreBundle to fronted independent module, it should be simpler to handle this in future :)